code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -131,6 +131,7 @@
 	..()
 	new /obj/item/clothing/mask/breath( src )
 	new /obj/item/weapon/reagent_containers/inhaler/phoron_special(src)
+	new /obj/item/weapon/reagent_containers/food/snacks/koisbar_clean(src)

```

Comment:
They already got their phoron inhaler, not sure why give vaurca more access to k'ois/phoron at roundstart, feels like this is going against stopping people from spawning with food suggestion.

Reply:
It basically is (hypocritical). If lunch boxes are power gaming by virtue of giving you access to an in-game resource you should normally work for, then this does exactly the same shit.

Reply:
I'll remove it then. I was on the fence about adding this in the first place.